README.md

```diff
@@ -0,0 +1,107 @@
+<a name="readme-top"></a>
+
+<h1>To Do List</h1>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+- [💻 Getting Started](#getting-started)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
```

Comment:
Kindly add the missing sections and links as requeted by the previous reviewer only FAQ is optional